README.md

```diff
@@ -63,7 +63,84 @@ Body:
     }
 ]
 ```
-
+### List one specific food by id in the database
+Request:
+```
+GET /api/v1/foods/:id
+Accept: application/json
+```
+Example response:
+```
+Status: 200
+Content-Type: application/json
+Body:
+{
+    "id": 1,
+    "name": "Banana",
+    "calories": 150
+}
+```
+### Create a new food
+Request:
+```
+POST /api/v1/foods
+Accept: application/json
+```
+Both name and calories are required fields.
+Pass in the following parameters:
+```
+{
+    "food": {
```

Comment:
I added an issue to resolve this (we can do it on a separate PR) -- we're not nesting within a `food` key right now.